src/Gui/OverlayManager.cpp

```diff
@@ -814,27 +814,27 @@ class OverlayManager::Private {
             return;
         case OverlayManager::OverlayMode::ToggleLeft:
             if (OverlayTabWidget::_LeftOverlay->isVisible())
-                OverlayTabWidget::_LeftOverlay->setState(OverlayTabWidget::State::Hidden);
+                OverlayTabWidget::_LeftOverlay->setState(OverlayTabWidget::State::Hidden, /*user_triggered*/true);

```

Comment:
If you made it an enum instead of a bool you could drop the comment.

Reply:
@chennes should I pick this, apply this change and drop new PR?

Reply:
IMO that would be delightful: or I can merge this and you can do it in another pass.